CIP-????/README.md

```diff
@@ -0,0 +1,220 @@
+
+### Serialization
+
+Transactions and blocks are serialized with
+[CBOR](https://www.rfc-editor.org/rfc/rfc7049)
+and specified with
+[CDDL file](https://www.rfc-editor.org/rfc/rfc8610).
+
+Serialization changes to the ledger are discussed in
+[CIP-80](https://github.com/cardano-foundation/CIPs/pull/372).
+
+Note that the serialisation format of the ledger state is unspecified and left as an
+implementation detail (unlike the format of blocks).
+
+### The ledger-script interface
+
+The ledger and Plutus scripts have a common interface, described in [CIP-35].
+CIPs relating to this inteface are relevant to both the ledger and to the Plutus CIP categories.
+
+Additionally, there is significant overlap with the Ledger category around the ledger-script
+interface and the protocol parameters.
+CIPs which change these parts of Cardano should generally use the Plutus category and not the
+Ledger category, although the Editors may ask the Ledger reviewers to comment.
+
+## Specification
+
+### What merits a ledger CIP?
+
+The criterion for deciding if a change to the ledger merits a CIP is as follows:
+changes to the ledger require going through the CIP process whenever
+every implementation of the Cardano ledger needs to be standardized on the details.
+
+Bug fixes are an exception to this criterion, they do not merit a CIP except in the case
+that the fix is substantially complicated.
+A "bug fix" is a change to behavior where:
+- The implemented behavior does not match the specification; or
+- The specified behavior is clearly wrong (in the judgment of relevant experts)
+
+Serialization changes are another possible exception to the criterion.
+Many serialization changes can be handled as a part of the normal development process
+without the need for a CIP.
+Dramatic changes to the serialization, however, may benefit from the CIP process.
+
+The ledger rules MUST be standardized in order for consensus to be maintained,
+but things like the ledger events are more open to debate.
+
+Changes to the protocol parameter values do not require a CIP since they are
+a governance issue (see [CIP-1694]).
+
+### Expectations for ledger CIPs
+
+* Familiarity with the existing ledger specifications is required to propose changes to the ledger.
+* The CIP specifications for ledger CIPs must be sufficiently detailed for inclusion in
+  a formal ledger specification.
+* Some changes may require getting an expert opinion.
+  For example, changes to the incentives model (such as changing the reward calculation)
+  should include a proper game theoretic analysis.
+
+### The Ledger reviewers
+
+The following table gives the current set of reviewers for Ledger CIPs.
+
+| Name          | Email                 | GitHub username |
+|---------------|-----------------------|-----------------|
+| Jared Corduan | jared.corduan@iohk.io | JaredCorduan    |
+| Andre Knispel | andre.knispel@iohk.io | WhatisRT        |
+
```

Comment:
I think the "Specification" would benefit from a short section about what it takes for a ledger CIP to _become active_. Are you expecting people to submit PRs to the ledger, or would you rather avoid people doing so (and have only the core maintainers do the work that's agreed on CIPs)?

Reply:
> Are you expecting people to submit PRs to the ledger, or would you rather avoid people doing so (and have only the core maintainers do the work that's agreed on CIPs)?

Something in the middle, I think. This involves perhaps a big topic that we should discuss.

Features do not get implemented in isolation in the ledger. We collect up a bunch of features, see how they interact, design a specification for a new ledger era, and then implement the era. I actually wrote up an earlier draft of this CIP that was much different than what I've submitted, where "ledger era" was its own ledger sub-type. In the end, I talked myself out of this since it could be argued that this is more of the release process. The ledger team would of course love help with PRs, but it would need to match our existing workflow, which is to build out the code once the era is settled. I'm not sure how the bundling should be involved in the CIP process.

Reply:
> short section about what it takes for a ledger CIP to become active

I could probably use some guidance from the CIP committee as to how to organize this. These are the stages in my mind:

* everyone agrees it is a good change
* it is selected for inclusion in some ledger era
* it is added to the formal spec
* it is implemented
* the hard fork happens and it is live